v1-4/mat_composition.ini

```diff
@@ -1,5 +1,5 @@
 
-mat fuel   -5.01 tmp 900  rgb 253 231 37 burn 1 vol 2.27785E+07 % vol 24.2496666m3
+mat fuel   -4.9602 tmp 900  burn 1 vol 1.9384117E-7 
```

Comment:
I am not sure about this density. You will need to determine the density for different temperatures. I think we should use the same correlation that Prof. Brooks. But I don't know which one they choose.

Reply:
I have a reference for this density in the README file but the operating temperature is out of the temperature range for the equation. I will email prof. Brooks and see if he has a better reference